sencha-workspace/SlateTasksStudent/app/controller/Dashboard.js

```diff
@@ -136,6 +148,49 @@ Ext.define('SlateTasksStudent.controller.Dashboard', {
         });
     },
 
+    onStudentSelectorSelect: function(combo, rec) {
```

Comment:
@jtwiest name these parameters `studentCombo, student`

Reply:
@themightychris I was following the lead set by `onSectionSelectorSelect` do you want me to update those param names as well?